airbyte-integrations/bases/standard-source-test/readme.md

```diff
@@ -3,31 +3,35 @@
 ## Overview
 
 ### These tests are designed to do the following:
+
 1. Test basic functionality for any Airbyte source. Think of it as an "It works!" test.
-    1. Each test should test functionality that is expected of all Airbyte sources.
+   1. Each test should test functionality that is expected of all Airbyte sources.
 1. Require minimum effort from the author of the integration.
-    1. Think of these are "free" tests that Airbyte provides to make sure the integration works.
+   1. Think of these are "free" tests that Airbyte provides to make sure the integration works.
 1. To be run for ALL Airbyte sources.
 
 ### These tests are _not_ designed to do the following:
+
 1. Test any integration-specific cases.
 1. Test corner cases for specific integrations.
 1. Replace good unit testing and integration testing for the integration.
 
 We will _not_ merge sources that cannot pass these tests unless the author can provide a very good reason™.
 
 ## What tests do the standard tests include?
+
 Check out each function in [SourceAcceptanceTest](src/main/java/io/airbyte/integrations/standardtest/source/SourceAcceptanceTest.java). Each function annotated with `@Test` is a single test. Each of these functions is proceeded by comments that document what they are testing.
 
 ## How to run them from your integration?
-* If writing a source in Python, check out this [readme](../base-python-test/readme.md)
+
+- If writing a source in Python, **don't use this.**
```

Comment:
🐍

Reply:
Wait... why have this line? is there a place we can link them too?